openapi/index.yaml

```diff
@@ -541,6 +541,42 @@ paths:
           description: Cannot regenerate service key.
           schema:
             $ref: "#/definitions/ProblemJson"
+  "/organizations/{organization_fiscal_code}/logo":
+    parameters:
+      - name: organization_fiscal_code
+        in: path
+        type: string
+        required: true
+        description: The Organization Fiscal Code.
+    put:
+      summary: Upload organization logo.
+      description: |
+        Upsert a logo for an Organization.
+        This API operation needs a valid API key
+        otherwise 403 forbidden will be returned to the caller.
+      operationId: uploadOrganizationLogo
+      parameters:
+        - name: body
+          in: body
+          required: true
+          schema:
+            $ref: "#/definitions/Logo"
+          description: A base64 string representation of the organization logo PNG image.
+      responses:
+        "200":
+          description: Logo uploaded.
+        "400":
+          description: Invalid payload.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+        "401":
+          description: Unauthorized.
+        "403":
+          description: Forbidden.
+        "500":
+          description: The service logo cannot be uploaded.
+          schema:
+            $ref: "#/definitions/ProblemJson"
```

Comment:
Maybe we want to add 404 in case no `Organization` is found with such fiscal code?

Reply:
Nope, there's another [story](https://www.pivotaltracker.com/story/show/175436161) to achieve this. At the moment we don't have a check mechanism cause Organization Model is missing.

Reply:
The point is that the handler can actually return 404